crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/AssessmentCriteriaServiceTest.java

```diff
@@ -166,7 +166,7 @@ public void givenPartnerFrequency_whenCheckAssessmentDetailIsInvoked_thenValidat
         String section = TestModelDataBuilder.TEST_SECTION;
         ApiAssessmentDetail detail = TestModelDataBuilder.getApiAssessmentDetails(true).get(0);
 
-        detail.setApplicantFrequency(null);
+        detail.setApplicantFrequency(Optional.ofNullable(null));
```

Comment:
`Optional.ofNullable(null)` not required. 
`detail.setApplicantFrequency(null);` should work

Reply:
As per SonarLint it was flagging, hence need to update.